### ISSUE TITLE
More login options

### ISSUE THREAD

**Issue by allytiago** [forem/forem#7087]
Okay, Twitter and Github are quite famous, but don't you think it would be better to bring new login options? Like, sometimes the member does not want to login by one of the above and ends up feeling obliged to create an account in one of the two to have to do something in dev.to. I believe it would be great to have login support via:
- Gitlab;
- Telegram, which has a great tool for external logins, just have a bot.
- Bitbucket

**Comment by pedrowindisch** [comment]
There have already been some other issues regarding adding other auth providers (like #1595 and #1865). I personally find the idea of another provider great, as I typically use my Google account to log in on websites.

**Comment by allytiago** [comment]
> There have already been some other issues regarding adding other auth providers (like #1595 and #1865). I personally find the idea of another provider great, as I typically use my Google account to log in on websites.

Perhaps it is not interesting to have Google login since it has a slightly distant meaning, which is different from Gitlab, Github, Twitter and Telegram which has many technology users.
I also think it would be nice to have these new login options for those who have neither github nor Twitter and had to create an account to use it. I myself am Gitlab user and have not used github since the period it was purchased.

**Comment by citizen428** [comment]
Thanks for the issue @allytiago! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
As a heads-up, we lately had some internal discussions around this, but can't yet make any promises regarding final scope or delivery date.

**Comment by jessleenyc** [comment]
Hey @allytiago - totally agree here. We're planning to add more auth options, we just need to do some refactoring first.

**Comment by allytiago** [comment]
> Hey @allytiago - totally agree here. We're planning to add more auth options, we just need to do some refactoring first.

I think it would be nice to add the possibility of login by git clients as an example gitlab. And since the site also acts as a web app and is functional on mobile devices, I think it would also be promising to have support for the Telegram auth.

**Comment by rhymes** [comment]
I'm going to close this as we've added Facebook since and we're in the process of adding Apple. The reason why I'm closing it's its genericness. I believe each request should sit in a specific issue so we can have separate conversations about feasibility and constraints
